epoxy-processor/src/main/java/com/airbnb/epoxy/ControllerModelField.kt

```diff
@@ -0,0 +1,36 @@
+package com.airbnb.epoxy
+
+import com.squareup.javapoet.TypeName
+
+import javax.lang.model.element.Element
+
+import com.airbnb.epoxy.Utils.isFieldPackagePrivate
+
+class ControllerModelField(element: Element) {
+
+    var fieldName: String = element.simpleName.toString()
+    var typeName: TypeName = TypeName.get(element.asType())
+    var packagePrivate: Boolean = isFieldPackagePrivate(element)
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
@geralt-encore cool, thanks!

btw, buck is on the verge of supporting Kotlin so we're very close to introducing it in our app. So, I've been studying up on it and we're trying to figure out which patterns to introduce to our team, and I'm curious to see how you do things :)